app/.env.local.example

```diff
@@ -1 +1,2 @@
-# BASE_API_URL="http://localhost:5000"
+BASE_API_URL="http://localhost:5000"
```

Comment:
do we need both? can't we just have 

`NEXT_PUBLIC_BASE_API_URL="http://localhost:5000"`

?

Reply:
yeah we could probably just use that one, you're right.